syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.fengjx.hello.grpc.proto";
option java_outer_classname = "HelloProto";
option objc_class_prefix = "HL";

package hello;

service GreetService {
    rpc sayHello (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string message = 1;
}
